var/spack/repos/builtin/packages/fortrilinos/package.py

```diff
@@ -24,21 +24,33 @@ class Fortrilinos(CMakePackage):
     """
 
     homepage = "https://trilinos.github.io/ForTrilinos/"
-    url      = "https://github.com/trilinos/ForTrilinos/archive/v2.0.0-dev1.tar.gz"
+    url      = "https://github.com/trilinos/ForTrilinos/archive/v2.0.0.tar.gz"
     git      = "https://github.com/trilinos/ForTrilinos.git"
 
     maintainers = ['sethrj', 'aprokop']
 
-    version('2.0.0-dev2', sha256='2a55c668b3fe986583658d272eab2dc076b291a5f2eb582a02602db86a32030b')
-    version('2.0.0-dev1', sha256='ab664ce2d7fe75c524d7ff6b1efffa3e459ab5739a916e6ea810ae40f39ca4f4')
+    version('2.0.0', sha256='9af3b3eea9934e44d74654a5fa822de08bd0efa43e06e4a4e35a777781f542d6')
+    # Note: spack version comparison implies Version('2.0.0') <
+    # Version('2.0.0-dev1'), so this is the best workaround I could find.
+    version('2.0.dev3',
+            sha256='c20a34b374a56b050bc1db0be1d3db63fca3e59c5803af0cb851b044ac84e6b3',
+            url="https://github.com/trilinos/ForTrilinos/archive/v2.0.0-dev3.tar.gz")
+    version('2.0.dev2',
+            sha256='2a55c668b3fe986583658d272eab2dc076b291a5f2eb582a02602db86a32030b',
+            url="https://github.com/trilinos/ForTrilinos/archive/v2.0.0-dev2.tar.gz")
+    version('2.0.dev1',
+            sha256='ab664ce2d7fe75c524d7ff6b1efffa3e459ab5739a916e6ea810ae40f39ca4f4',
+            url="https://github.com/trilinos/ForTrilinos/archive/v2.0.0-dev1.tar.gz")
     version('master', branch='master')
 
     variant('hl', default=True, description='Build high-level Trilinos wrappers')
     variant('shared', default=True, description='Build shared libraries')
 
     # Trilinos version dependencies
-    depends_on('trilinos@12.18.1', when='@2.0.0-dev2')
-    depends_on('trilinos@12.17.1', when='@2.0.0-dev1')
+    depends_on('trilinos@13', when='@2.0.0:')
+    depends_on('trilinos@12.18.1', when='@2.0-dev3')
+    depends_on('trilinos@12.18.1', when='@2.0-dev2')
+    depends_on('trilinos@12.17.1', when='@2.0-dev1')
```

Comment:
```suggestion
    depends_on('trilinos@12.18.1', when='@2.0.dev3')
    depends_on('trilinos@12.18.1', when='@2.0.dev2')
    depends_on('trilinos@12.17.1', when='@2.0.dev1')
```

Reply:
Oops not sure how that slipped by. Good catch.